docs/dashboards/dashboards-overview.asciidoc

```diff
@@ -6,12 +6,14 @@
 :frontmatter-tags-content-type: [overview]
 :frontmatter-tags-user-goals: [visualize, monitor, analyze]
 
-The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
+This section documents the {security-app}'s default dashboards, which provide visualizations of your security environment. To view them in {security-app}, select *Dashboards* from the navigation menu.
+From the Dashboards page, you can access the default dashboards, as well as create and access custom dashboards.
 
-You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Custom dashboards must have the tag `Security Solution` to appear on the Dashboards page.
+To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by using the text search field to filter by name and description, or by using the *Tags* menu to filter by tag. You can also click one of a custom dashboard's tags to toggle whether to filter for that tag. To create a new tag, open the *Tags* menu and click *Manage tags*.
```

Comment:
```suggestion
To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by:
* Using the text search field to filter by name or description
* Using the **Tags** menu to filter by tag
* Clicking one of a custom dashboard's tags to toggle whether to filter for that tag
To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.
```
A couple of tiny suggestions, otherwise LGTM!

(The reason I suggested replacing single asterisks with double is to reduce the potential amount of [fixes after the migration to MDX](https://github.com/elastic/staging-serverless-security-docs/pull/49#discussion_r1297444042).)